sdk/src/invariant.ts

```diff
@@ -408,8 +409,19 @@ export class Invariant {
   async getTick(poolKey: PoolKey, index: bigint): Promise<Tick> {
     const stateRootHash = await this.client.nodeClient.getStateRootHash()
     const buffor: number[] = []
-    const indexBytes = bigintToByteArray(index)
-    const preparedIndexBytes = indexBytes.concat(Array(4 - indexBytes.length).fill(0))
+
+    const indexBytes =
```

Comment:
IMO, even if this code will only be used once, it's worth moving it to utilities function to improve readability. Also, check if this isn't somewhere in the wasm.